word_detector/__init__.py

```diff
@@ -119,6 +119,7 @@ def _cluster_lines(detections: List[DetectorRes],
             union = a.h + b.h - intersection
             iou = np.clip(intersection / union if union > 0 else 0, 0, 1)
             dist_mat[i, j] = 1 - iou  # Jaccard distance is defined as 1-iou
```

Comment:
please write `dist_mat[i, j] = dist_mat[j, i] = 1 - iou` here to get the full matrix. I don't want to rely on internals of the DBSCAN implementation.